main.js

```diff
@@ -16,9 +16,12 @@ function direction(event) {
     case 'ArrowRight':
       $car.className = 'east';
       break;
-    case ' ':
-      intervalID = setInterval(driveCar, 16);
-      break;
+  }
+  if (key === ' ' && intervalID === null) {
+    intervalID = setInterval(driveCar, 16);
+  } else {
+    clearInterval(intervalID);
+    intervalID = null;
```

Comment:
I think this can still be a `case` in your `switch` statement for the `" "` key, you will just need a conditional in the work to determine if an interval is already in progress